installers/msi-language/CustomFatalError.wxs

```diff
@@ -12,17 +12,34 @@
 				<Control Id="Back" Type="PushButton" X="180" Y="243" Width="56" Height="17" Disabled="yes" Text="!(loc.WixUIBack)" />
 				<Control Id="BottomLine" Type="Line" X="0" Y="234" Width="370" Height="0" />
 				<Control Id="Title" Type="Text" X="135" Y="20" Width="220" Height="60" Transparent="yes" NoPrefix="yes" Text="!(loc.FatalErrorTitle)" />
-				<Control Id="Description1" Type="Text" X="135" Y="70" Width="220" Height="40" Transparent="yes" NoPrefix="yes" Text="!(loc.FatalErrorDescription1)" />
-				<Control Id="CustomText" Type="Hyperlink" X="135" Y="110" Width="220" Height="20" Transparent="yes" Hidden="no" Text="!(loc.FatalErrorPrimary)" />
-				<Control Id="Description2" Type="Text" X="135" Y="140" Width="220" Height="20" Transparent="yes" NoPrefix="yes" Text="!(loc.FatalErrorDescription2)" />
+				<Control Id="Description1" Type="Text" X="135" Y="70" Width="220" Height="40" Transparent="yes" Hidden="yes" NoPrefix="yes" Text="!(loc.FatalErrorDescription1)">
+					<Condition Action="show">NETWORK_ERROR = "false"</Condition>
+				</Control>
+				<Control Id="CustomTextFatalError" Type="Hyperlink" X="135" Y="110" Width="220" Height="20" Transparent="yes" Hidden="yes" Text="!(loc.FatalErrorPrimary)">
+					<Condition Action="show">NETWORK_ERROR = "false"</Condition>
+				</Control>
+				<Control Id="CustomTextNetworkError" Type="Hyperlink" X="135" Y="70" Width="220" Height="60" Transparent="yes" Hidden="yes" Text="!(loc.NetworkErrorPrimary)">
+					<Condition Action="show">NETWORK_ERROR = "true"</Condition>
+				</Control>
+				<Control Id="CustomTextNetworkErrorDetails" Type="Hyperlink" X="135" Y="130" Width="220" Height="40" Transparent="yes" Hidden="yes" Text="!(loc.NetworkErrorDetails)">
```

Comment:
nit: This could also be of type text (unless URLs show up as hyperlink if they are part of the error message).